### ISSUE TITLE
If touchScrollTarget doesn't have a scrollbar, the page can still be scrolled

### ISSUE THREAD

**Issue by Dremora** [jossmac/react-scrolllock#34]
I'm not even sure if there is any (easy) solution, but I'll post this anyway.

It seems that `touchScrollTarget` should be scrollable in order for `touchmove` events not to propagate to the body. However, in our app we don't always know if the contents fits into scrollable container or not. If it does, scroll events will scroll the `touchScrollTarget`. However, in case it doesn't, they will scroll `<body>`.

**Comment by jossmac** [comment]
@Dremora this is a bit hard to follow. Could you provide a reproduction using [CodeSandbox](http://codesandbox.io/)?

**Comment by DrChai** [comment]
+1, @jossmac here is reproduction https://codesandbox.io/s/nn1vlq9lmj

**Comment by Dremora** [comment]
@theKashey in your example, the modal itself is not scrollable. Reduce the browser window height and try scrolling.

**Comment by Dremora** [comment]
@xemexpress `body { overflow-y: hidden; }` doesn't work on iOS.

**Comment by theKashey** [comment]
@Dremora - you mean - it is not scrollable as it __should__. There is nothing to "allow" it.
"ScrollLock" could only remove scrolling, not add it.
See https://codesandbox.io/s/2po5mm8qz0 - I've set `max-height` to content, thus __browser__ would add scroll bar to a modal content.

**Comment by theKashey** [comment]
They are almost the same, the trick like this - https://github.com/jossmac/react-scrolllock/issues/33#issuecomment-437744062 - leverages all the difference.

**Comment by jossmac** [comment]
@theKashey what are you up to there buddy?

**Comment by theKashey** [comment]
🤷‍♂️ as I remember - I've posted it to point that `It seems that touchScrollTarget should be scrollable` is not true.
